### PR TITLE
updates for working with next svd2rust release

### PR DESCRIPTION
**Pull request by burrbull**
https://github.com/rust-embedded/svd2rust/pull/508

### CODE REVIEW

**Comment by burrbull** [comment]
cc @adamgreig 

So, I see a lot of `bitWidth` bugs.
```
[ERROR svd2rust] In device `STM32G0B0`
    
    Caused by:
        0: In peripheral `TIM14`
        1: In register `CCMR1_Output`
        2: Parsing field #4
        3: In field `OC1M2`
        4: Value 2 out of range 0..2
```

**Comment by burrbull** [comment]
Rebased on #546 

Require https://github.com/stm32-rs/svdtools/pull/57 !!!

**Comment by burrbull** [comment]
@adamgreig rebased

**Comment by adamgreig** [comment]
Thanks for working through this! I need to check some of the G0 ones a bit more carefully, but otherwise the main question is about all the F7 and L4 `tim_ccm_v2.yaml` files that are commented out: what needs to be done with them?

File: peripherals/flash/flash_latency16.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      WS6: [6, "6 wait states"]
+      WS7: [7, "7 wait states"]
+      WS8: [8, "8 wait states"]
+      WS9: [9, "9 wait states"]
+      WS10: [10, "10 wait states"]
+      WS11: [11, "11 wait states"]
+      WS12: [12, "12 wait states"]
+      WS13: [13, "13 wait states"]
+      WS14: [14, "14 wait states"]
+      WS15: [15, "15 wait states"]
```
Comment (adamgreig) [inline peripherals/flash/flash_latency16.yaml:19]: Do we even need this latency16 file? Looks like all users of flash_v1.yaml use the flash_latency8.yaml version.

**Comment by adamgreig** [comment]
I've finished reviewing and I guess it makes sense to merge this before tackling #546 since it's based off it. My only outstanding question really is about the commented-out CCMR files, what do you think?

* Make new CCMR YAMLs for the different field widths and apply them as required to each device?
* Merge this now and do that in another PR?
* Something else?

**Comment by burrbull** [comment]
> I've finished reviewing and I guess it makes sense to merge this before tackling #546 since it's based off it. My only outstanding question really is about the commented-out CCMR files, what do you think?
> 
>     * Make new CCMR YAMLs for the different field widths and apply them as required to each device?
> 
>     * Merge this now and do that in another PR?
> 
>     * Something else?

The comments of  CCMR files is fast fix to make compile, as currently those files are broken.
A real fix of this you can see in #546. So I propose merge this. And then review #546.

File: peripherals/flash/flash_latency16.yaml (hunk 1 above)
Comment (burrbull) [reply peripherals/flash/flash_latency16.yaml:19]: Looks like I planned to do something, but forgot.

Comment (burrbull) [reply peripherals/flash/flash_latency16.yaml:19]: According to RM only 405/415 407/417 series in STM32F4 don't support higher states.

Comment (burrbull) [reply peripherals/flash/flash_latency16.yaml:19]: Fixed

**Review by adamgreig — APPROVED** [review]
Thank you very much for working this out!

Understood about CCR being fixed in #546.

I've just noticed on a second review that the F7x2's rebase of TIM5 to use TIM2 means its option register is no longer correct, but let's fix that in another PR. Everything else LGTM, thanks for fixing the flash latency bits.

bors merge

**Pull request merged**